bullet-featherstone/src/EntityManagementFeatures.cc

```diff
@@ -206,53 +206,16 @@ Identity EntityManagementFeatures::ConstructEmptyWorld(
 bool EntityManagementFeatures::RemoveModel(const Identity &_modelID)
 {
   auto *model = this->ReferenceInterface<ModelInfo>(_modelID);
-  auto *world = this->ReferenceInterface<WorldInfo>(model->world);
-  if (world->modelIndexToEntityId.erase(model->indexInWorld) == 0)
-  {
-    // The model has already been removed at some point.
+  if (!model)
     return false;
```

Comment:
removed. [ca1715e](https://github.com/gazebosim/gz-physics/pull/574/commits/ca1715e2a41ecb070fe8d81d7530f75e0c985339)

Reply:
well, now I think I was wrong, since I didn't notice that we are dereferencing `model` on the next line. Those two lines from https://github.com/gazebosim/gz-physics/pull/574/commits/ca1715e2a41ecb070fe8d81d7530f75e0c985339 should probably come back. Thanks for your patience

Reply:
ah yes, reverted change. [d08d0ef](https://github.com/gazebosim/gz-physics/pull/574/commits/d08d0ef83a4e4cc638ff11eb792877be90a0ea9f)